example/storybook/src/components/Feedback/Spinner/index.stories.mdx

```diff
@@ -98,9 +98,9 @@ The Examples section provides visual representations of the different variants o
       code: `
         <HStack space="md" >
       <Spinner size="small"/>
-      <Heading  fontSize="xs">
+      <Text  size="md">
```

Comment:
Please use fontSize prop instead of size

Reply:
okay